deps/chakrashim/include/v8-debug.h

```diff
@@ -91,7 +91,7 @@ class V8_EXPORT Debug {
   static Local<Context> GetDebugContext(Isolate* isolate);
 
   static void EnableDebug();
-  static void EnableInspector();
+  static void EnableInspector(bool enableReplayDebug = false);
```

Comment:
>EnableInspector [](start = 14, length = 15)

This function is not in deps/v8/include/v8-debug.h generally we keep V8 header same between v8 and chakrashim. Is this specific to chakrashim?